its-a-smol-world/requirements.txt

```
@@ -0,0 +1,3 @@
+outlines==0.1.0
```

```suggestion
outlines==0.1.1
```

Might be able to go up to 0.1.1